src/actions/actions/PaymentMethod/WyreAccount.js

```diff
@@ -41,21 +43,23 @@ const getPaymentAddress = (account, currency) => {
     }
 };
 
-export const manageAccount = (navigation) => async(dispatch, getState) => {
+export const manageAccount = (navigation) => async (dispatch, getState) => {
     const state = getState();
     const paymentMethod = selectWyrePaymentMethod(state);
-    if (!paymentMethod) {
+
+    if (!hasPaymentMethod(paymentMethod)) {
         // Create account
         try {
             dispatch(createWyreAccount());
+            const activeAccount = selectActiveAccount(state);
 
-            const { data, key, error } = await WyreService.build().createAccount();
+            const { data, key, error } = await WyreService.build().createAccount(paymentMethod.key);
 
             dispatch(createWyreAccountResponse());
 
             if (error) return DelayedAlert('Failed creating Wyre account');
 
-            const activeAccount = selectActiveAccount(state);
+            await storeData(activeAccount.id, { userId: data.id, key: paymentMethod.key });
```

Comment:
We are not going to store this information in the store. 
The only viable secret is the wifKey and that is already in the store.
We want this information in the redux/memory store,. Please check if this line is doing that
dispatch(signIntoAccount(newActiveAccount));